Libplanet.Stun/Stun/TurnClient.cs

```diff
@@ -124,26 +124,28 @@ public class TurnClient : IStunContext, IDisposable
                     await _connectionAttempts.DequeueAsync(cancellationToken);
 
                 byte[] id = attempt.ConnectionId;
+                var bindRequest = new ConnectionBindRequest(id);
                 var relayedClient = new TcpClient(_host, _port);
                 NetworkStream relayedStream = relayedClient.GetStream();
 
-                var bindRequest = new ConnectionBindRequest(id);
-                await SendMessageAsync(
-                    relayedStream,
-                    bindRequest,
-                    cancellationToken);
-                StunMessage bindResponse =
-                    await StunMessage.Parse(relayedStream);
-
-                if (bindResponse is ConnectionBindSuccessResponse)
+                try
                 {
-                    _relayedClients.Add(relayedClient);
-                    return relayedStream;
-                }
+                    await SendMessageAsync(relayedStream, bindRequest, cancellationToken);
+                    StunMessage bindResponse = await StunMessage.Parse(relayedStream);
 
-                throw new TurnClientException(
-                        "ConnectionBind failed.",
-                        bindResponse);
+                    if (bindResponse is ConnectionBindSuccessResponse)
+                    {
+                        _relayedClients.Add(relayedClient);
+                        return relayedStream;
+                    }
+
+                    throw new TurnClientException("ConnectionBind failed.", bindResponse);
+                }
+                catch (IOException e)
+                {
+                    Log.Warning(e, "The connection seems to disconnect before parsing. ignored.");
+                    continue;
```

Comment:
If disconnected, shouldn't we `break` instead of `continue`?

Reply:
It's intended because we should contiue to handle other new connections even if we caught an exception from previous connectio .